src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+public class Rectangle extends Shape {
+    private double height;
+    private double width;
+
+    public Rectangle(String color, double height, double width) {
+        super(Colors.valueOf(color));
+        this.height = Math.round(Math.abs(height));
+        this.width = Math.round(Math.abs(width));
+
+    }
+
+    public double getHeight() {
+        return height;
+    }
+
+    public void setHeight(double height) {
+        this.height = height;
```

Comment:
Consider validating the `height` value in the `setHeight` method to ensure it is non-negative, similar to the constructor.